Podwodny świat tabliczki mnożenia/.collabignore

```diff
@@ -0,0 +1,33 @@
+# ===========================
```

Comment:
As I wrote at your last pull request, do you see this filename? It includes polish chars and spaces :(